src/models/DefaultData.js

```diff
@@ -173,20 +168,17 @@ class DefaultData {
 
   /**
    * Generate id from definition and components list.
-   * @param {ComponentDefinition} definition - Component definition.
-   * @returns {string} String that is the concatenation of the definition type and an index.
+   * The id is composed of "id_" and the number of component plus 1.
+   * @returns {string} Unique string for the id.
    */
-  generateComponentId(definition) {
-    const templateId = `${definition.type}_`;
+  generateComponentId() {
+    const templateId = 'id_';
     const ids = this.components
       .map(({ id }) => id)
       .filter((id) => new RegExp(`${templateId}\\d+`).test(id))
       .map((id) => parseInt(id.substring(templateId.length), 10));
-    let index = 1;
 
-    while (ids.includes(index)) {
-      index += 1;
-    }
+    const index = ids.length === 0 ? 1 : ids.pop() + 1;
```

Comment:
prefere to have an auto-increment like max id +1

Reply:
done